docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md

```diff
@@ -0,0 +1,122 @@
+
+## Prerequisites
+
+The Asset Hub parachain is one of the system parachains on a relay chain, such as [Polkadot](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank} or [Kusama](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank}. To interact with these parachains, you can use the [Polkadot.js App](https://polkadot.js.org/apps/#/explorer){target=\_blank} interface. For example:
+
+- [Polkadot Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/explorer){target=\_blank}
+- [Kusama Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fsys.ibp.network%2Fstatemine#/explorer){target=\_blank}
+
+For testing purposes, you can also interact with the Asset Hub instance of test networks:
+
+- [Rococo Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-rococo-rpc.dwellir.com#/explorer){target=\_blank}
+- [Paseo Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpas-rpc.stakeworld.io%2Fassethub#/explorer){target=\_blank}
+
+Before you start, ensure that you have: 
+
+- Access to the Polkadot.js App interface, and you are connected to the desired chain
+- A parachain that supports the XCMP protocol to interact with the Asset Hub parachain
+- A funded wallet to pay for the transaction fees and subsequent registration of the foreign asset
+
+This guide will focus on using Polkadot, its local Asset Hub instance, and the [Astar](https://astar.network/){taget=\_blank} parachain (`ID` 2006), as stated in the [Test Environment Setup](./register-a-foreign-asset.md/#test-enviroment-setup) section. However, the process is the same for other relay chains and their respective Asset Hub parachains, regardless of the network you are using and the parachain owner of the foreign asset.
+
+## Steps to Register a Foreign Asset
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the `Asset Hub` parachain
+
+      - For the local environment, connect to Local Node (Chopsticks), check the [Test Environment Setup](./register-a-foreign-asset.md/#test-enviroment-setup) section for more details
+      - For the live networks, connect to the `Asset Hub` parachain. You can use either Polkadot, Kusama, Rococo, or Paseo Asset Hub
+
+2. Click on the **Developer** tab on the left sidebar and select the **Extrinsics** section
+
+    ![Access to Developer Extrinsics section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-1.webp)
+
+3. Select the **Foreign Asset** pallet from the dropdown list and choose the **create** extrinsic
+
+    ![Select the Foreign Asset pallet](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-2.webp)
```

Comment:
The arrows in this image are smaller than in other images

Reply:
Will re-take all screenshots according to the discussion we had yesterday